packages/lib/services/response.ts

```diff
@@ -2,10 +2,33 @@ import { prisma } from "@formbricks/database";
 import { TResponse, TResponseInput, TResponseUpdateInput } from "@formbricks/types/v1/responses";
 import { Prisma } from "@prisma/client";
 import { DatabaseError, ResourceNotFoundError } from "@formbricks/errors";
-import { transformPrismaPerson } from "./person";
+import { TransformPersonOutput, transformPrismaPerson } from "./person";
 
 export const createResponse = async (responseInput: TResponseInput): Promise<TResponse> => {
   try {
+    let transformedPerson: TransformPersonOutput | null = null;
+
+    if (responseInput.personId) {
+      const person = await prisma.person.findUnique({
```

Comment:
We are currently transforming our infrastructure to rely more on services for data retrieval. I think here it would be better to use the person service `getPerson()` from `../person.ts`.